test/unit/gateways/payu_latam_test.rb

```diff
@@ -187,9 +187,9 @@ def test_partial_buyer_hash_info
         phone: "(11)756312345"
       ),
       buyer: {
-        name: 'Jorge Borges',
-        dni_number: '5415668464456',
-        email: 'axaxaxas@mlo.org'
+        name: "Jorge Borges",
```

Comment:
You didn't actually change these lines, but they're flagged due to quote changes.